tests/test-sources/plugins/by-name/showkeys/default.nix

```
@@ -0,0 +1,59 @@
+{ lib, ... }:
+{
+  default = {
```

```suggestion
  empty = {
```
Nit. Otherwise LGTM!